credsweeper/app.py

```diff
@@ -4,12 +4,12 @@
 import signal
 import sys
 from pathlib import Path
-from typing import Any, List, Optional, Union
+from typing import Any, List, Optional, Union, Dict
 
 import pandas as pd
 
 from credsweeper.app_path import APP_PATH
-from credsweeper.common.constants import KeyValidationOption, ThresholdPreset
+from credsweeper.common.constants import KeyValidationOption, ThresholdPreset, Severity
```

Comment:
```suggestion
from credsweeper.common.constants import KeyValidationOption, Severity, ThresholdPreset
```

Reply:
What reason of the swap? Alphabet order only?